packages/ui-patterns/form/FormLayout2.tsx

```diff
@@ -28,7 +28,7 @@ import {
 } from 'ui'
 import { z } from 'zod'
 import { Input } from '../DataInputs/Input'
-import { InfoTooltip } from '../InfoTooltip/InfoTooltip'
+import { InfoTooltip } from '../info-tooltip'
```

Comment:
Capitalized and CamelCased `InfoTooltip.tsx` to keep component file names consistent?

Reply:
will come back to this later